hack/verify-gofmt.sh

```diff
@@ -23,10 +23,15 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-kube::golang::verify_go_version
-
 cd "${KUBE_ROOT}"
 
+# Prefer bazel's gofmt.
+gofmt="external/io_bazel_rules_go_toolchain/bin/gofmt"
```

Comment:
this makes me a little sad, but I'm not sure if there's a better way (besides adding this to the PATH?)